aui.views/src/AUI/Platform/win32/CommonRenderingContext.cpp

```diff
@@ -52,8 +52,8 @@ void CommonRenderingContext::init(const Init& init) {
         winClass.hInstance = GetModuleHandle(nullptr);
         //winClass.hIcon = LoadIcon(mInst, (LPCTSTR)101);
         //winClass.hIconSm = LoadIcon(mInst, (LPCTSTR)101);
-        winClass.hIcon = 0;
-        winClass.hIconSm = 0;
+        winClass.hIcon = LoadIcon(winClass.hInstance, L"MAINICON");
+        winClass.hIconSm = LoadIcon(winClass.hInstance, L"MAINICON");
```

Comment:
Maybe do `LoadIcon` only once?
```suggestion
        static const auto mainIcon = LoadIcon(winClass.hInstance, L"MAINICON");
        winClass.hIcon = mainIcon;
        winClass.hIconSm = mainIcon;
```